Framework/LiveData/src/Kafka/private/Schema/y2gw_run_info_generated.h

```diff
@@ -0,0 +1,367 @@
+// automatically generated by the FlatBuffers compiler, do not modify
```

Comment:
I thought you'd need to disable clang-format here. I guess you've let it format this file? I don't think it is worth changing now, but we try not to modify the generated headers so that they can easily be compared.